.github/workflows/update-checksums-file.yml

```diff
@@ -0,0 +1,49 @@
+name: 'Update Wrapper checksums file'
+
+on:
+  schedule:
+    # Run weekly (at arbitrary time)
+    - cron: '24 5 * * 6'
+  # Support running workflow manually
+  workflow_dispatch:
+
+jobs:
+  update-checksums:
+    name: Update checksums
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: 20.x
+          cache: npm
+
+      - name: Install dependencies
+        run: |
+          npm install typed-rest-client@1.8.11 --no-save
+
+      - name: Update checksums file
+        run: node ./.github/workflows/update-checksums-file.js
+
+      # If there are no changes, this action will not create a pull request
+      - name: Create or update pull request
+        uses: peter-evans/create-pull-request@v6
+        with:
+          branch: wrapper-checksums-update
```

Comment:
Maybe the branch name should use a prefix such as `bot/` to prevent accidentally manually editing this branch, which might then be overwritten by this workflow.

What do you think?

Reply:
Sounds completely reasonable to prefix it with `bot/`